trillian/ctfe/configpb/config.proto

```diff
@@ -154,3 +182,26 @@ message SignedTreeHead {
   bytes sha256_root_hash = 3;
   bytes tree_head_signature = 4;
 }
+
+// Cache describes the cache type and configurations.
+//
+// NEXT_ID: 5
+message Cache {
+  // CacheType represents the underlying cache type implementation.
+  // If unspecified, no cache is used.
+  enum CacheType {
+    CACHE_TYPE_NONE = 0;
+    CACHE_TYPE_LRU = 1;
+  }
+  CacheType type = 1;
+
+  // Cache size 0 means unlimited size. Default is 0.
+  int64 size = 2;
+
+  // Cache ttl 0 means cache will never expire. Default is 0.
```

Comment:
Same here, a reasonably high value like 12h or 1d shouldn't hurt, but might prevent unfortunate memory growth. I don't think you can do much in the code for these zero values as it depends on the library implementation really but it might be worth mentioning in the docs.

Reply:
The cache size is much more predictable than the TTL. I expect most CT log operators will make use of the cache use. Most cache implementation provides TTL, so it is reasonable to provide the TTL setting here.